### PR TITLE
Update root files urls for github workflows

### PR DESCRIPTION
**Pull request by kkacprzak**
The automated tests with the git hub workflows are not working, because the files with the root packages disappeared from the website. Updating the run scheme with the newest root versions available on the downloads page.

### CODE REVIEW

File: .github/workflows/ci-workflow.yml [hunk 1]
```diff
@@ -28,8 +28,8 @@ jobs:
       if: matrix.os == 'ubuntu-20.04'
     - name: download  ROOT (for ubuntu-18.04)
```
Comment (wkrzemien) [inline .github/workflows/ci-workflow.yml:29]: For consistency, you should change the description for the correct ubuntu version, the same above

File: .github/workflows/ci-workflow.yml [hunk 2]
```diff
@@ -14,12 +14,12 @@ jobs:
     - name: system update
       run: |
        sudo apt-get -qq update
-       sudo apt-get install -y git libboost-all-dev libtbb-dev cmake
+       sudo apt-get install -y git libboost-all-dev libtbb-dev cmake nlohmann-json3-dev
 
     - name: download  ROOT (for ubuntu 22.04)
```
Comment (wkrzemien) [inline .github/workflows/ci-workflow.yml:19]: change the description, please

Comment (wkrzemien) [reply .github/workflows/ci-workflow.yml:19]: Sorry my bad, It is root version that has changed not the ubuntu. Please ignore it

**Review by wkrzemien — APPROVED** [review]

**Comment by kkacprzak** [comment]
The configuration for ubuntu 22 and 20 worked, the job for ubuntu 18 just won't start. I have tried several times, always with the message "waiting for the runner to pick up the job".

**Pull request merged**